atom_tools/lib/converter.py

```diff
@@ -558,8 +553,10 @@ def _filter_matches(self, matches: List[str], code: str) -> List[str]:
 
         for m in matches:
             if m and m[0] not in ('.', '@', ','):
-                nm = m.replace('"', '').replace("'", '').lstrip('/')
-                filtered_matches.append(f'/{nm}')
+                nm = m.replace('"', '').replace("'", '')
+                # Checking for / cuts down lots of false positives
+                if nm not in ("*", "*/*"):
```

Comment:
@prabhu I have added an exclusion list for things like this in my filtering branch.